homeassistant/components/deconz/services.yaml

```
@@ -1,32 +1,71 @@
 configure:
-  description: Set attribute of device in deCONZ. See https://home-assistant.io/integrations/deconz/#device-services for details.
+  name: Configure
+  description: >-
+    Configure attributes of either a device endpoint in deCONZ
+    or the deCONZ service itself.
   fields:
     entity:
-      description: Entity id representing a specific device in deCONZ.
+      name: Entity
+      required: false
+      description: Represents a specific device endpoint in deCONZ.
       example: "light.rgb_light"
+      selector:
+        entity:
+          integration: deconz
     field:
+      name: Path
+      required: false
+      selector:
+        text:
       description: >-
-        Field is a string representing a full path to deCONZ endpoint (when
+        String representing a full path to deCONZ endpoint (when
         entity is not specified) or a subpath of the device path for the
         entity (when entity is specified).
       example: '"/lights/1/state" or "/state"'
     data:
-      description: Data is a JSON object with what data you want to alter.
+      name: Configuration payload
+      required: true
+      selector:
+        text:
+      description: JSON object with what data you want to alter.
       example: '{"on": true}'
     bridgeid:
-      description: (Optional) Bridgeid is a string unique for each deCONZ hardware. It can be found as part of the integration name.
+      name: Bridge identifier
+      required: false
+      selector:
+        text:
```

object to get code mirror?

```suggestion
        object:
```